docs/csharp/programming-guide/concepts/linq/basic-linq-query-operations.md

```diff
@@ -95,7 +95,7 @@ from order in Customer.Orders...
   
 ## See also
 
-- [Getting Started with LINQ in C#](../../../../csharp/programming-guide/concepts/linq/getting-started-with-linq.md)
+- [Getting Started with LINQ in C#](../../../../csharp/programming-guide/concepts/linq/index.md)
```

Comment:
I think that the original link was here just to fulfill the requirement to link to the parent. The actual content of the new link target doesn't mirror a getting started, which is really what this whole section is about. I think that I'd just remove the link or rename it to something like **Language Integrated Query (LINQ) overview**.
,